libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/TwoPhaseValidation.hs

```diff
@@ -1955,25 +1966,38 @@ testUTXOWsubset,
       PostShelley era
     ) =>
     WitRule "UTXOW" era ->
+    UTxO era ->
     Core.PParams era ->
     Core.Tx era ->
     Either [(PredicateFailure (Core.EraRule "UTXOW" era))] (State (Core.EraRule "UTXOW" era)) ->
     Assertion
 
 -- | Use an equality test on the expected and computed [PredicateFailure]
-testUTXOW wit@(UTXOW (Alonzo _)) = testUTXOWwith wit genericCont
-testUTXOW wit@(UTXOW (Babbage _)) = testUTXOWwith wit genericCont
-testUTXOW (UTXOW other) = error ("Cannot use testUTXOW in era " ++ show other)
+testUTXOW wit@(UTXOW (Alonzo _)) utxo = testUTXOWwith wit genericCont utxo
+testUTXOW wit@(UTXOW (Babbage _)) utxo = testUTXOWwith wit genericCont utxo
+testUTXOW (UTXOW other) _ = error ("Cannot use testUTXOW in era " ++ show other)
```

Comment:
Why did you eta expanded it? Seems unnecessary.

Reply:
I have no idea why I did that, must be vestigial, good catch!